src/libusermode/userhook.hpp

```diff
@@ -113,6 +115,17 @@
 #include "plugins/plugins_ex.h"
 #include "printers/printers.hpp"
 
+class UserHookException : public std::exception
+{
+public:
+    UserHookException(std::string&& msg) : msg{msg} {}
+    virtual const char* what() const noexcept
+    {
+        return this->msg.c_str();
+    }
+    std::string msg;
+};
```

Comment:
Kinda :donotwant: as it does same as `std::runtime_error`

Reply:
k